app/modules/asset_groups/models.py

```diff
@@ -829,34 +778,6 @@ def get_sage_job_status(cls, job_id):
             'job.status', 'get', {}, job_id
         )
 
-    # Build up list to print out status (calling function chooses what to collect and print)
```

Comment:
I'd argue that this is useful debug inforamtion and unless this is replaced entirely (including showing the response) it should be retained, or at least some way of a developer accessing the job information from the AGS guid without having t navigate via the progress.